src/libcore/macros.rs

```diff
@@ -821,4 +823,43 @@ mod builtin {
         ($cond:expr,) => ({ /* compiler built-in */ });
         ($cond:expr, $($arg:tt)+) => ({ /* compiler built-in */ });
     }
+
+    macro_rules! doit {
+        ($($name:ident)*) => (
+            $(
+                /// Doc
+                #[rustc_doc_only_macro]
+                #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
This could use some comments re. why e.g. asm! is stable...?

Reply:
Sure, I'll do it later, right now only the infrastructure is up, but the specific data is not filled in yet.